datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -1785,29 +1785,42 @@ select min(t), max(t) from  (select '00:00:00' as t union select '00:00:01' unio
 ----
 00:00:00 00:00:02
 
-# aggregate_decimal_min
-query RT
-select min(c1), arrow_typeof(min(c1)) from d_table
+# aggregate Interval(MonthDayNano) min/max
```

Comment:
I think these tests needs to be updated now that you have implemented the correct logic

You can do so by running 

```shell
cargo test --test sqllogictests  -- aggregate --complete
```

And then verifying and checking in the results

Reply:
updated; I think they look right at a glance, but I'm not well versed in this part of the system. can you verify?

Reply:
They looked good to me -- thanks!